scriptworker/constants.py

```diff
@@ -165,6 +165,8 @@
         # gecko-focus is for mozilla-mobile releases. It's named this way because it's a worker type
         # but using the gecko ChainOfTrust keys. See bug 1455290 for more details.
         "gecko-focus",
+        # staging releases can run on the regular github-worker.
+        "github-worker",
```

Comment:
To be honest, I'm concerned about this line. If I remember correctly, "gecko-1-decision" was created because we want segregation between taskcluster workers that anybody can use and workers only level-1 people can use. I wonder if we should do something similar for github.

Reply:
I think maybe we want to have a configuration like `verify_cot_signature` that conditionally (i.e. for staging workers) either doesn't check this list, or checks against a more permissive

Reply:
Discussed with Aki over IRC. One clean solution would be to create `mobile-{1..3}-decision worker types. This way, we don't allow `github-worker` anymore. We're using a more trusted subset of workers. 

https://bugzilla.mozilla.org/show_bug.cgi?id=1512631